bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/DecimalType.java

```diff
@@ -165,6 +165,9 @@ public State as(Class<? extends State> target) {
             } else {
                 return UnDefType.UNDEF;
             }
+        } else if (target == HSBType.class) {
+            return new HSBType(DecimalType.ZERO, PercentType.ZERO,
+                    new PercentType(this.toBigDecimal().multiply(new BigDecimal(100))));

```

Comment:
`BigDecimal.valueOf(100)` is preferred to `new BigDecimal(100)`:  "This 'static factory method' is provided in preference to a (long) constructor because it allows for reuse of frequently used BigDecimal values." ([link](https://docs.oracle.com/javase/7/docs/api/java/math/BigDecimal.html#valueOf(long)))